src/pdnssoccli/pdnssoccli.py

```diff
@@ -17,7 +17,8 @@ def configure(ctx, param, filename):
     try:
         with open(filename) as config_file:
             parsed_config = yaml.safe_load(config_file)
-    except:
+    except Exception as e:
```

Comment:
We should refrain from using print for error logging. For this reason there is already a `logger` object declared. I suggest replacing this except block:


```suggestion
    except FileNotFoundError as e:
        logger.error("Configuration file {} not found. Exiting...".format(filename))
        exit(1)
    except:
        logger.error("Unexpected error while opening {} configuration file. Exiting...".format(filename))
        exit(1)
```

Reply:
Sounds good!